cloud.ganttproject.colloboque/src/main/kotlin/cloud/ganttproject/colloboque/Postgres.kt

```diff
@@ -57,11 +54,11 @@ class PostgresDataSourceFactory(
         stmt.execute()
       }
     }
-    return HikariDataSource(HikariConfig().apply {
-      username = pgSuperUser
-      password = pgSuperAuth
-      jdbcUrl = "jdbc:postgresql://${pgHost}:${pgPort}/dev_all_projects"
-      this.schema = schema
-    }).also { cachedDataSources[projectRefid] = it }
   }
+
+  fun createConnection(projectRefid: String): Connection =
+    superDataSource.connection.also { it.schema = getSchema(projectRefid) }
+
+  // TODO: escape projectRefid
+  private fun getSchema(projectRefid: String) = "project_$projectRefid"
```

Comment:
refid in our reality is a pretty long string. I suggest hashing it like this:

`Hashing.murmur3_128().hashBytes(refid).toString()`